site/content/en/docs/administration/advanced/upgrade_guide.md

```diff
@@ -56,6 +56,12 @@ To upgrade CVAT, follow these steps:
   docker logs cvat_server -f
   ```
 
+## Upgrade CVAT after v2.25.0
+
+In version 2.25.0, CVAT changed the location where the export cache is stored.
+To clean up the outdated cache, run the following command: `python manage.py exportcachecleanup`.
+
```

Comment:
> Enqueuing RQ jobs to backup a project/task: A filename argument was removed from the background function signature. Previously enqueued jobs will fail.

What do you think about adding a one-off migration script to avoid losing requested RQ jobs? It should probably just remove 1 parameter from the existing jobs.

Reply:
In that case, we'll have to make this PR dependent on https://github.com/cvat-ai/cvat/pull/8898

Reply:
I'm not sure that this is a real problem. I don't think there will be many such RQ jobs with `Queued` status when upgrading to the next release.  Additionally, I don't think that someone uses frequently the django admin panel to re-enqueue the job manually. In other cases, everything should work.
But, if @SpecLad and @zhiltsov-max are voting to add such a migration (e.g. as an example of the first Redis migration, I can do that)

Reply:
Probably, we should also extract these documentation changes into a separate PR that will be merged on the same day of release.

Reply:
I'm not voting for that (and I also don't want to make this PR dependent on another one...).

I also doubt that it's a big problem, and even if it is, it seems like there's a much simpler solution - just add the parameter back (and ignore it).